app/models/manageiq/providers/ibm_power_vc/cloud_manager.rb

```diff
@@ -413,6 +413,7 @@ def self.params_for_create
                     :id        => "endpoints.node.options",
                     :name      => "endpoints.node.options",
                     :label     => _("Resource File Path"),
+                    :hideField => true,
```

Comment:
LGMT, but we'll need to merge https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/339 first in order to get a fixed RC file path, right?
https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/337/files#diff-51b83da8bac79fa3e8f2fa891bed6165c8255531bde333fa4379f90617638dafR80

Reply:
Yep, ManageIQ/manageiq-providers-ibm_cloud#337 (tried to link to the specific line I was looking at).